src/libtest/lib.rs

```diff
@@ -1482,20 +1483,26 @@ pub fn run_test(opts: &TestOpts,
         }
         DynTestFn(f) => {
             let cb = move |()| {
-                __rust_begin_short_backtrace(|| f.call_box(()))
+                begin_short_backtrace(|| f.call_box(()))
             };
             run_test_inner(desc, monitor_ch, opts.nocapture, Box::new(cb))
         }
         StaticTestFn(f) =>
             run_test_inner(desc, monitor_ch, opts.nocapture,
-                           Box::new(move |()| __rust_begin_short_backtrace(f))),
+                           Box::new(move |()| begin_short_backtrace(f))),
     }
 }
 
-/// Fixed frame used to clean the backtrace with `RUST_BACKTRACE=1`.
-#[inline(never)]
-fn __rust_begin_short_backtrace<F: FnOnce()>(f: F) {
-    f()
+/// Clean the backtrace by using std::rt::mark_backtrace_start
+#[inline(always)]
```

Comment:
Does this need to be either `#[inline]` or `#[inline(always)]`? If so, can it be documented as to why?

Reply:
I don't think the attribute is needed here.